crl-janitor/pubsub.tf

```diff
@@ -0,0 +1,43 @@
+resource "google_pubsub_topic" "crl-janitor-pubsub-topic" {
+  count = var.enable ? 1 : 0
+
+
+  project = var.google_project
+  name = "${local.owner}-${local.service}-pubsub-topic"
+}
+
+resource "google_pubsub_subscription" "crl-janitor-pubsub-subscription" {
+  count = var.enable ? 1 : 0
+
+
+  project = var.google_project
+  name = "${local.owner}-${local.service}-pubsub-sub"
+  topic = google_pubsub_topic.crl-janitor-pubsub-topic.name
+
+  ack_deadline_seconds = 600
```

Comment:
10 minutes is pretty long. Maybe 20 or 30 seconds? If our janitor hasn't acked by then, it's probably not going to.

Reply:
changed to 20

Reply:
20s SG